README.MD

```diff
@@ -10,13 +10,13 @@ GitHub allows you to [co-author commits](https://help.github.com/articles/creati
 
 ## 🧠 Advanced
 
-### Flags
+### Make it blazing fast 🔥
 
-You can also utilize git flags with co-commit: eg `git commit --all` becomes `npx co-commit --all`. Additionally, you can skip prompts using flags: eg: `-m "your commit message"` or `-co "mariiapunda, juliettepretot"`.
+Each time you run co-commit, `npx` will install the package. This will usually only takes a few seconds. To skip this you can preinstall co-commit using `npm i -g co-commit`.
 
-### Preinstall co-commit
+### CLI arguments
 
-Each time you run co-commit, `npx` will install the package. This will usually take around one second. To skip this you can preinstall co-commit using `npm i -g co-commit`.
+You can also utilize git flags with co-commit: eg `git commit --all` becomes `npx co-commit --all`. Additionally, you can skip prompts using flags: eg: `-m "your commit message"` or `-co "mariiapunda"`.
```

Comment:
`npx co-commit mariiapunda` would actually be awesome UX

Reply:
Fair point!